<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Authentication</class>
 <widget class="QMainWindow" name="Authentication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Authentication</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/images/iconRegistration.jpg</normaloff>:/images/images/iconRegistration.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {

border-image: url(:/images/backgroundAunthetication.jpg);

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>361</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Login</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="labelLogin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>STIX</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineLogin">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>181</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="linePassword">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>181</width>
       <height>32</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelPassword">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>STIX</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRegister">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>88</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>STIX</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonLogin">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>88</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>STIX</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
